README.md

```diff
@@ -1,64 +1,228 @@
+
 ## Contributing
 
-Please read CONTRIBUTING.md guide to know more.
+Please read [CONTRIBUTING.md](https://github.com/metakgp/gyft/blob/master/CONTRIBUTING.md) guide to know more.
 
 ## Maintainer(s)
 
 - [Ashwin Prasanth](https://github.com/ashwinpra)
+<!-- CONTACT -->
+<!-- ## Contact -->
+
+<!-- <p> -->
+<!-- 📫 Arpit Bhardwaj ( aka proffapt ) - -->
+<!-- <a href="https://twitter.com/proffapt">
+  <img align="center" alt="proffapt's Twitter " width="22px" src="https://raw.githubusercontent.com/edent/SuperTinyIcons/master/images/svg/twitter.svg" />
+</a>
+<a href="https://t.me/proffapt">
+  <img align="center" alt="proffapt's Telegram" width="22px" src="https://raw.githubusercontent.com/edent/SuperTinyIcons/master/images/svg/telegram.svg" />
+</a>
+<a href="https://www.linkedin.com/in/proffapt/">
+  <img align="center" alt="proffapt's LinkedIn" width="22px" src="https://raw.githubusercontent.com/edent/SuperTinyIcons/master/images/svg/linkedin.svg" />
+</a>
+<a href="mailto:proffapt@pm.me">
+  <img align="center" alt="proffapt's mail" width="22px" src="https://raw.githubusercontent.com/edent/SuperTinyIcons/master/images/svg/mail.svg" />
+</a>
+<a href="https://cybernity.group">
+  <img align="center" alt="proffapt's forum for cybernity" width="22px" src="https://cybernity.group/uploads/default/original/1X/a8338f86bbbedd39701c85d5f32cf3d817c04c27.png" />
+</a>
+</p>
+
+<p align="right">(<a href="#top">back to top</a>)</p> -->
+
+
+<!-- ACKNOWLEDGMENTS -->
+<!-- ## Acknowledgments
+
+* [Choose an Open Source License](https://choosealicense.com)
+* [Img Shields](https://shields.io)
+<!-- UPDATE -->
+<!-- 
+<p align="right">(<a href="#top">back to top</a>)</p> -->
+
+<!-- ## Additional documentation
+
+  - [Changelogs](/.github/CHANGELOG.md)
+  - [License](/LICENSE.txt)
+  - [Security Policy](/.github/SECURITY.md)
+  - [Code of Conduct](/.github/CODE_OF_CONDUCT.md)
+  - [Contribution Guidelines](/.github/CONTRIBUTING.md)
+
+<p align="right">(<a href="#top">back to top</a>)</p> -->
+
+<!-- MARKDOWN LINKS & IMAGES -->
+
+[contributors-shield]: https://img.shields.io/github/contributors/metakgp/gyft.svg?style=flat
+[contributors-url]: https://github.com/metakgp/gyft/graphs/contributors
+[forks-shield]: https://img.shields.io/github/forks/metakgp/gyft.svg?style=flat
+[forks-url]: https://github.com/metakgp/gyft/network/members
+[stars-shield]: https://img.shields.io/github/stars/metakgp/gyft.svg?style=flat
+[stars-url]: https://github.com/metakgp/gyft/stargazers
+[issues-shield]: https://img.shields.io/github/issues/metakgp/gyft.svg?style=flat
+[issues-url]: https://github.com/metakgp/gyft/issues
+[license-shield]: https://img.shields.io/github/license/metakgp/gyft.svg?style=flat
+[license-url]: https://github.com/metakgp/gyft/blob/master/LICENSE.txt
+[linkedin-shield]: https://img.shields.io/badge/-LinkedIn-black.svg?style=flat&logo=linkedin&colorB=555
+[linkedin-url]: https://linkedin.com/in/proffapt
```

Comment:
Replace these with similar things for wiki and redirecting to wiki.metakgp.org

Reply:
Also use `style=for-the-badge`